doc/tor.1.txt

```diff
@@ -111,50 +111,55 @@ debugging messages to **`debug.log`**, you must specify **`--Log
     is as specified in version-spec.txt.)
 
 [[opt-quiet]] **--quiet**|**--hush**::
-    Override the default console log.  By default, Tor starts out logging
-    messages at level "notice" and higher to the console.  It stops doing so
-    after it parses its configuration, if the configuration tells it to log
-    anywhere else.  You can override this behavior with the **--hush** option,
-    which tells Tor to only send warnings and errors to the console, or with
-    the **--quiet** option, which tells Tor not to log to the console at all.
+    Override the default console logging behavior.  By default, Tor
+    starts out logging messages at level "notice" and higher to the
+    console.  It stops doing so after it parses its configuration, if
+    the configuration tells it to log anywhere else.  These options
+    override the default console logging behavior.  Use the **--hush**
+    option if you want Tor to log only warnings and errors to the
+    console, or use the **--quiet** option if you want Tor not to log
+    to the console at all.
 
 [[opt-keygen]] **--keygen** [**--newpass**]::
-   Running "tor --keygen" creates a new ed25519 master identity key for a
-   relay, or only a fresh temporary signing key and certificate, if you
-   already have a master key.  Optionally you can encrypt the master identity
-   key with a passphrase: Tor will ask you for one. If you don't want to
-   encrypt the master key, just don't enter any passphrase when asked. +
- +
-   The **--newpass** option should be used with --keygen only when you need
-   to add, change, or remove a passphrase on an existing ed25519 master
-   identity key. You will be prompted for the old passphase (if any),
-   and the new passphrase (if any). +
- +
-   When generating a master key, you will probably want to use
-   **--DataDirectory** to control where the keys
-   and certificates will be stored, and **--SigningKeyLifetime** to
-   control their lifetimes.  Their behavior is as documented in the
-   server options section below.  (You must have write access to the specified
-   DataDirectory.) +
- +
-   To use the generated files, you must copy them to the DataDirectory/keys
-   directory of your Tor daemon, and make sure that they are owned by the
-   user actually running the Tor daemon on your system.
+    Running "tor --keygen" creates a new ed25519 master identity key
+    for a relay, or only a fresh temporary signing key and
+    certificate, if you already have a master key.  Optionally, you
+    can encrypt the master identity key with a passphrase.  When Tor
+    asks you for a passphrase and you don't want to encrypt the master
+    key, just don't enter any passphrase when asked. +
+ +
+    Use the **--newpass** option with --keygen only when you need to
+    add, change, or remove a passphrase on an existing ed25519 master
+    identity key. You will be prompted for the old passphase (if any),
+    and the new passphrase (if any).
++
```

Comment:
I notice this isn't indented any more. Is that intentional? My asciidoc skills are rusty.

Reply:
Yes. The `+` on a line by itself is a list continuation, unlike the `+` at the end of a line, which was a line break. The list continuation makes the following paragraph part of the preceding list item. If I put the `+` on a line by itself as a list continuation, the indented paragraph gets rendered as a literal paragraph (which is more visible in the HTML output compared to the manpage output) instead of as a normal paragraph.

On the other hand, it seems like by explicitly putting paragraph styles like `[NOTE]` and `[normal]` in front of the continuation paragraphs, the indentation no longer results in the paragraph rendering as a literal paragraph. I can make that change if you think it helps to keep the paragraphs indented in the Asciidoc source.

Reply:
I think it's fine; I trust your asciidoc.